MODULE.bazel

```diff
@@ -6,25 +6,25 @@ module(
     compatibility_level = 1,
 )
 
-bazel_dep(name = "bazel_features", version = "1.15.0")
-bazel_dep(name = "bazel_skylib", version = "1.3.0")
-bazel_dep(name = "platforms", version = "0.0.5")
-bazel_dep(name = "rules_python", version = "0.23.1")
+bazel_dep(name = "bazel_features", version = "1.21.0")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "rules_python", version = "1.0.0")
```

Comment:
I'd suggest that we update rules_python to 1.0.0 as a individual PR as there's a few things to be fixed up when doing this.. I'd also like to update the WORKSPACE version to be in sync so we're not targetting 2 different APIs in rules_python..